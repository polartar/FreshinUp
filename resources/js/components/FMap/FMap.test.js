import Component from './FMap'
import { shallowMount } from '@vue/test-utils'

describe('components/FMap', () => {
  // no snapshots tests since it will break on CI

  describe('Props & Computed', () => {
    test('accessToken', async () => {
      const wrapper = shallowMount(Component, {
        propsData: {
          accessToken: 'aaa111'
        }
      })
      expect(wrapper.vm.accessToken).toEqual('aaa111')

      wrapper.setProps({
        accessToken: 'bbb222'
      })
      expect(wrapper.vm.accessToken).toEqual('bbb222')
    })
    test('mStyle', async () => {
      const wrapper = shallowMount(Component)
      expect(wrapper.vm.mStyle).toEqual('mapbox://styles/mapbox/streets-v11')

      const mStyle = 'mapbox://styles/mapbox/streets-v12'
      wrapper.setProps({
        mStyle
      })
      expect(wrapper.vm.mStyle).toEqual(mStyle)
    })
  })
})
