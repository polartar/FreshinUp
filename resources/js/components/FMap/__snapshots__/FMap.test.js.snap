// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/FMap Snapshots Basic 1`] = `
<div
  class="mgl-map-wrapper"
>
  <div
    id="map-9404048319742673"
  />
  <!---->
</div>
`;
